### PR TITLE
OSPool Ganglia: lock the container to el8 to get a comptabile ganglia/php combo

### PR DESCRIPTION
**Pull request by rynge**

### CODE REVIEW

**Comment by brianhlin** [comment]
Does EL9 work? Because if you use `BASE_OS` like so https://github.com/opensciencegrid/images/blob/main/opensciencegrid/cvmfs-nodeplugin/Dockerfile, you'll get OSG 23 + EL9 images and OSG 3.6 + EL7 images

**Comment by rynge** [comment]
No, el9 is the broken one. It ships with php8 which does not work with Ganglia.

File: opensciencegrid/ospool-ganglia/Dockerfile [hunk 1]
```diff
@@ -1,22 +1,48 @@
...
       ganglia-gmetad \
+      ganglia-gmond \
       ganglia-web \
+      git \
       httpd \
+      lsof \
       php-fpm \
+      python3-pip \
+      rrdtool \
+      vim \
```
Comment (brianhlin) [inline opensciencegrid/ospool-ganglia/Dockerfile:28]: Really? This seems unnecessary...

File: opensciencegrid/ospool-ganglia/Dockerfile [hunk 2]
```diff
@@ -1,22 +1,48 @@
...
+      wget \
```
Comment (brianhlin) [inline opensciencegrid/ospool-ganglia/Dockerfile:29]: What's this for?

File: opensciencegrid/ospool-ganglia/Dockerfile [hunk 3]
```diff
@@ -1,22 +1,48 @@
...
 ARG BASE_YUM_REPO
 
-RUN dnf -y install \
-      ganglia-gmond \
+# has to be redefined for use in the RUN stages
+ARG BASE_YUM_REPO
+
+RUN dnf -y distro-sync && \
+    dnf -y install \
+      epel-release && \
```
Comment (brianhlin) [inline opensciencegrid/ospool-ganglia/Dockerfile:16]: EPEL is already included in the base image

**Review by brianhlin — APPROVED** [review]
Software is the worst. Preapproving

File: opensciencegrid/ospool-ganglia/Dockerfile (hunk 2 above)
Comment (rynge) [reply opensciencegrid/ospool-ganglia/Dockerfile:29]: Just mimicing the ospool-cm image.

**Pull request merged**